.clang-format

```diff
@@ -0,0 +1,94 @@
+# SPDX-FileCopyrightText: 2020 CERN
+# SPDX-License-Identifier: Apache-2.0
+---
+Language:        Cpp
+# BasedOnStyle:  LLVM
+AccessModifierOffset: -2
+AlignAfterOpenBracket: Align
+AlignConsecutiveAssignments: true
+# This would be nice to have but seems to also (mis)align function parameters
```

Comment:
This comment leads me to wonder if an updated clang-format would work better (I'm not 100% sure what the problem aluded to is). Maybe @agheata remembers?